index.js

```diff
@@ -11,6 +11,7 @@ import interpret from './interpreter.js';
 
 import { commands } from './orderedCommands.js';
 import { colours, WHITE, BLACK } from './colours.js';
+import Jimp from "jimp";
```

Comment:
Not sure if we need that import and the package or if your parent repo provides it?